jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamDebugSupport.kt

```diff
@@ -60,7 +38,7 @@ interface SamDebugSupport {
         val promise = AsyncPromise<XDebugProcessStarter?>()
         val bgContext = ExpirableExecutor.on(AppExecutorUtil.getAppExecutorService()).expireWith(environment).coroutineDispatchingContext()
 
-        val timerTask = Timer("Debugger Worker launch timer", true).schedule(debuggerAttachTimeoutMs) {
+        val timerTask = Timer("Debugger Worker launch timer", true).schedule(SamDebugger.debuggerConnectTimeoutMs()) {
```

Comment:
maybe a TODO, but i thought there was some talk about resetting the timer on process output, like VSC

Reply:
That would require a lot of additional changes, not sure if we actually want to do that but definitely not part of this PR